h2o-docs/src/product/data-science/algo-params/upload_custom_metric.rst

```diff
@@ -3,7 +3,7 @@
 ``upload_custom_metric``
 ------------------------
 
-- Available in: GBM, DRF
+- Available in: GBM, DRF, Deeplearning
```

Comment:
I'm wondering if `upload_custom_metric` should be "available" for the algos. Algos use it and hopefully by the end of these PRs all the supervised algos will support it. Uploading a resource is in my opinion unrelated to the usage in the algos (if not, should we have "availability" for `import_file`?).

It's more of a question than a suggestion to change it.